sass/breadcrumbs.scss

```diff
@@ -1,25 +1,32 @@
-.castlecss-breadcrumbs{
-	
-	.breadcrumb-wrap {
-		display: block;		
+.castlecss-breadcrumbs {
```

Comment:
@amberjasmijn Thanks for your reply, the prefix will be removed in the next version.